### PR TITLE
Places api updates

### PR DESCRIPTION
**Pull request by domesticmouse**
PTAL @samthor @markmcd 

DO NOT SUBMIT kthnxsbi =)

### CODE REVIEW

File: places.go [hunk 1]
```diff
@@ -28,6 +29,122 @@ import (
...
+	Language string
+	// MinPrice restricts results to only those places within the specified price level. Valid values are in the range from 0 (most affordable) to 4 (most expensive), inclusive.
+	MinPrice PriceLevel
+	// MaxPrice restricts results to only those places within the specified price level. Valid values are in the range from 0 (most affordable) to 4 (most expensive), inclusive.
+	MaxPrice PriceLevel
+	// Name is one or more terms to be matched against the names of places, separated with a space character.
+	Name string
+	// OpenNow returns only those places that are open for business at the time the query is sent. Places that do not specify opening hours in the Google Places database will not be returned if you include this parameter in your query.
+	OpenNow bool
+	// Rankby specifies the order in which results are listed.
```
Comment (samthor) [inline places.go]: Maybe `RankBy`?

File: places.go [hunk 2]
```diff
@@ -115,10 +236,110 @@ type TextSearchRequest struct {
...
+
+func (r *RadarSearchRequest) params() url.Values {
+	q := make(url.Values)
+
+	if r.Location != nil {
+		q.Set("location", r.Location.String())
+	}
+
+	if r.Radius != 0 {
+		q.Set("radius", fmt.Sprint(r.Radius))
```
Comment (samthor) [inline places.go]: can you just `string()` this?

File: types.go [hunk 3]
```diff
@@ -141,3 +145,332 @@ type Photo struct {
...
+const (
+	RankbyProminence = Rankby("prominence")
+	RankbyDistance   = Rankby("distance")
+)
+
+// PlaceType TODO(brettmorgan)
+type PlaceType string
+
+// Place Types for the Places API
+const (
```
Comment (samthor) [inline types.go]: do we have a nice way to autogenerate this if it changes?

Comment (domesticmouse) [inline types.go]: I have some code that has templates for generating both the Golang and Java versions of this enumeration. It is in a private repo at this point.

File: places.go (hunk 1 above)
Comment (domesticmouse) [inline places.go]: Done

File: places.go (hunk 2 above)
Comment (domesticmouse) [inline places.go]: amusingly string() on a uint winds up generating this: radius=%E2%9C%90

**Comment by domesticmouse** [comment]
PTAL @samthor - we are now ready to publish this.

**Comment by samthor** [comment]
LGTM

**Comment by domesticmouse** [comment]
Committing a squashed merge from a new PR.

**Pull request closed without merging**